### PR TITLE
4-get-api-reviews

### PR DESCRIPTION
**Pull request by tyr-ros**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -61,4 +61,46 @@ describe('GET: /api', () => {
...
+        it("returns a 404 if there is no review with the id", () => {
+            return request(app)
+                .get('/api/reviews/1099')
+                .expect(404)
+                .then(({ body }) => {
+                    expect(body.msg).toBe('Bad request')
+
+
+                })
+        })
```
Comment (Keb84) [inline __tests__/app.test.js:104]: Another test for an invalid ID would also be good here 🔍 (for example, the ID is a string)

**Review by Keb84 — CHANGES_REQUESTED** [review]
Nicely done Rebecca. Please add the test suggested and resubmit 🙂

File: __tests__/app.test.js (hunk 1 above)
Comment (tyr-ros) [reply __tests__/app.test.js:104]: Will do!

**Review by MrMcSnail — APPROVED** [review]
Good work. Merge at will 🏇 🥳

**Pull request merged**